AnnotatoBackend/Sources/App/Persistence/SharedLibraryModels+Persistable/Document+Persistable.swift

```diff
@@ -1,12 +1,19 @@
+import Foundation
 import AnnotatoSharedLibrary
 
 extension Document: Persistable {
     static func fromManagedEntity(_ managedEntity: DocumentEntity) -> Self {
-        Self(
+        let document = Self(
+            id: managedEntity.id,
             name: managedEntity.name,
             ownerId: managedEntity.ownerId,
-            baseFileUrl: managedEntity.baseFileUrl,
-            id: managedEntity.id
+            baseFileUrl: managedEntity.baseFileUrl
         )
+
+        managedEntity.annotations.forEach { annotationEntity in
+            document.addAnnotation(annotation: Annotation.fromManagedEntity(annotationEntity))
+        }
```

Comment:
```suggestion
        document.annotations = managedEntity.annotations.map(Annotation.fromManagedEntity);
```

Would this be cleaner? Then we won't need the `addAnnotation` function.

Reply:
This would require the setter to be public

Reply:
Ah okay, what about doing this in the init call? Is there a reason why it must be done after the init call currently